dbmongo/js/reduce.algo2/algo2_tests.ts

```diff
@@ -105,3 +107,56 @@ function sortObject(object: any): any {
 
   return sortedObj
 }
+
+test("delai_deviation_remboursement est calculé si un délai de règlement de cotisations sociales a été demandé", (t: ExecutionContext) => {
+  const dateDebut = new Date("2018-01-01")
+  const datePlusUnMois = new Date("2018-02-01")
+  initGlobalParams(dateDebut, datePlusUnMois)
+  const siret = "012345678901234"
```

Comment:
```suggestion
  const siret = "12345678901234"
```
Un caractère trop long ;) (même si ça change rien, donc prend peut-être pas le risque d'accepter !)